sqlite.lua-scm-1.rockspec

```diff
@@ -0,0 +1,37 @@
+rockspec_format = "3.0"
+package = 'sqlite.lua'
+version = 'scm-1'
+description = {
+  detailed = "",
+  homepage = "https://github.com/tami5/sqlite.lua",
+  labels = { "sqlite3", "binding", "luajit", "database" },
+  license = "MIT",
+  summary = "SQLite/LuaJIT binding and a highly opinionated wrapper for storing, retrieving, caching, and persisting [SQLite] databases"
+}
+source = {
+  url = 'git://github.com/tami5/sqlite.lua.git',
+  tag = "master"
+}
+dependencies = {
+  "plenary.nvim"
```

Comment:
Oh wait, sqlite.lua doesn't depend on plenary only on luv, have you seen script/gen_rockspec.lua

Reply:
I think I had seen it then forgot about it when I resumed the PR xD I've submitted an update that adds plenary as a test dependency though. It won't work because plenary is not in the root manifest of luarocks.org but I think this can be done with the help of @tjdevries

Reply:
```
luarocks test ./sqlite-master-0.rockspec 
Missing dependencies for sqlite master-0:
   plenary.nvim (not installed)

sqlite master-0 depends on plenary.nvim (not installed)
Installing https://luarocks.org/plenary.nvim-scm-1.rockspec
```
fails . 
adding my manifest fixes it (https://luarocks.org/modules/teto/plenary.nvim) but it is not scalable so I wonder if I should manage plenary myself or if @tjdevries has enough time/interest to do it (or another maintainer) .

I think it's fine to merge as is as `luarocks install ./sqlite-master-0.rockspec  --tree /tmp` keeps working

Reply:
I've just pushed a linting fix. I think it would be best to have the CI use the flake to run the code rather than CI actions. We would have the same results locally as on github + it would work on other CI/CD pipelines without limiting oneself to github. but that's a suggestion.

Reply:
Yes, the CI setup was pre getting into nix. this would make things a lot simpler and won't have this out sync issue.